packages/core/test/unit/segments/attributes/trace_id.test.js

```diff
@@ -0,0 +1,47 @@
+var assert = require('chai').assert;
+var TraceID = require('../../../../lib/segments/attributes/trace_id');
+
+function validateTraceID(traceID) {
```

Comment:
Can we write a test when there is overlong arbitrary field in traceheader  ? Ex. Root=1-58406520-a006649127e371903a2de979;Parent=53995c3f42cd8ad8;Sampled=1;Foo=Bar;[A field bigger than 256 bytes]

Reply:
@bhautikpip Good call. I added this functionality & tests [here](https://github.com/aws/aws-xray-sdk-node/pull/284/files#diff-6a36c6c7818cffd493bc93a85aebe9b4R92-R111)!